### PR TITLE
Lesson 7

### PR DESCRIPTION
**Pull request by studentNV**
@vauboy comment me

### CODE REVIEW

File: lesson13-14/home_work13-14.md [hunk 1]
```diff
@@ -0,0 +1,517 @@
...
+
+#### 4. Check the root directory space usage by du command. Compare it with an information from df. If you find differences, try to find out why it happens.
+```bash
+[sitis@localhost lesson7]$ df -h /
+Filesystem               Size  Used Avail Use% Mounted on
+/dev/mapper/centos-root  6.2G  2.0G  4.3G  33% /
+[sitis@localhost lesson7]$ du -sh /
+2.1G    /
+```
+> The difference is that du queries directly for every file found in the partition, while df queries the filesystem. When a file is deleted, which at that moment was “occupied” by a process, its name is deleted, but the inode remains in the file system until the process that “holds” this file ends.
```
Comment (vauboy) [inline lesson13-14/home_work13-14.md:460]: Not exactly. Current task is connected with the previous one.

Comment (studentNV) [reply lesson13-14/home_work13-14.md:460]: @vauboy I didn't quite understand you. Maybe the answer is different number metrics?

Comment (vauboy) [reply lesson13-14/home_work13-14.md:460]: Could you check du's argument '-x' meaning?

Comment (vauboy) [reply lesson13-14/home_work13-14.md:460]: I think it will be clear enough to understand, so you can just merge the changes

Comment (studentNV) [reply lesson13-14/home_work13-14.md:460]: @vauboy I understand correctly now I can merge?

**Pull request merged**